### PR TITLE
Switch out the default env vars now that stitching has been working fine for a few weeks

### PR DESCRIPTION
**Pull request by orta**
This means PRs won't need rebasing because someone doesn't have the right local env vars set up, will comment inline.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -60,7 +59,3 @@ REQUEST_THROTTLE_MS=5000
 REQUEST_TIMEOUT_MS=5000
 RESIZING_SERVICE=gemini
 RESOLVER_TIMEOUT_MS=0
-
-ENABLE_SCHEMA_STITCHING=true
-ENABLE_GRAVQL_ONLY_STITCHING=true
-ENABLE_KAWS_STITCHING=true
```
Comment (orta) [inline .env.example]: ENABLE_SCHEMA_STITCHING is now inverses: `DISABLE_SCHEMA_STITCHING`

Kaws & GravQL are now stitched by default.

File: _schema.graphql [hunk 2]
```diff
@@ -4489,9 +4489,10 @@ type Metaphysics {
   queryTracing: Boolean!
   heapDumps: Boolean!
   stitching: Boolean!
-  stitchingGravity: Boolean!
   stitchingConvection: Boolean!
   stitchingExchange: Boolean!
+  stitchingGravity: Boolean!
+  stitchingKaws: Boolean!
 }
```
Comment (orta) [inline _schema.graphql]: Expands the GraphQL MP config schema to note that kaws is also happening

File: scripts/compare-stitched-schema.sh [hunk 3]
```diff
@@ -9,6 +9,6 @@ fi
 
 mkdir -p tmp
 yarn dump-schema _schema.graphql
-ENABLE_SCHEMA_STITCHING=true ENABLE_GRAVQL_ONLY_STITCHING=true yarn dump-schema tmp/stitched.graphql
+yarn dump-schema tmp/stitched.graphql
 
 schema_comparator compare  _schema.graphql tmp/stitched.graphql
```
Comment (orta) [inline scripts/compare-stitched-schema.sh]: This file is mainly used for debugging stitching, will need to change it when working with exchange

File: src/config.ts [hunk 4]
```diff
@@ -91,11 +92,15 @@ const mustHave = {
   IMPULSE_APPLICATION_ID,
   POSITRON_API_BASE,
   EXCHANGE_API_BASE,
+  KAWS_API_BASE,
```
Comment (orta) [inline src/config.ts]: This puts it in `mustHave` which means tests/app won't load unless you've set it, and will give a useful error message

File: src/config.ts [hunk 5]
```diff
@@ -91,11 +92,15 @@ const mustHave = {
...
 }
 
 Object.keys(mustHave).forEach(key => {
   if (!mustHave[key]) {
-    throw new Error(`You need to have the ENV var ${key} set up.`)
+    const file = chalk.whiteBright(".env.example")
+    throw new Error(
+      `You need to have the ENV var ${key} set up - check out ${file}.`
+    )
   }
```
Comment (orta) [inline src/config.ts]: notes where to find the var

File: .env.example (hunk 1 above)
Comment (alloy) [reply .env.example]: Should there be a commented out example of how to disable it so in a moment of crisis people can find it here?

File: _schema.graphql (hunk 2 above)
Comment (alloy) [reply _schema.graphql]: Oh didn’t know about this piece of introspection 👍 (What’s `environment: Boolean!`?)

File: src/schema/me/recently_viewed_artworks.ts [hunk 6]
```diff
@@ -30,33 +30,3 @@ export const RecentlyViewedArtworks = {
...
-    if (!recordArtworkViewLoader) {
-      throw new Error(
-        "Missing recordArtworkViewLoader. Check that `X-Access-Token` and `X-User-Id` headers are set."
-      )
-    }
-    return recordArtworkViewLoader({ artwork_id }).then(() => {
-      return { artwork_id }
-    })
-  },
-})
```
Comment (alloy) [inline src/schema/me/recently_viewed_artworks.ts]: This is in GravQL, right?

File: src/schema/services/metaphysics.ts [hunk 7]
```diff
@@ -8,10 +8,11 @@ const mapEnvBooleans = {
   environment: !!config.NODE_ENV,
```
Comment (alloy) [inline src/schema/services/metaphysics.ts]: Re my earlier question, what does it mean to a client for `NODE_ENV` to be set or not?

File: src/test/utils.ts [hunk 8]
```diff
@@ -61,6 +61,8 @@ export const runAuthenticatedQuery = (
 
 let mergedSchema
 
+// TODO: runQueryMerged could be removed now?
```
Comment (alloy) [inline src/test/utils.ts]: I guess so. Is it clear how to mock data for stitched links?

**Comment by alloy** [comment]
Looking good, some small comments.

File: src/schema/services/metaphysics.ts (hunk 7 above)
Comment (orta) [reply src/schema/services/metaphysics.ts]: This is a good point, I'll switch this over to a string

File: src/schema/me/recently_viewed_artworks.ts (hunk 6 above)
Comment (orta) [reply src/schema/me/recently_viewed_artworks.ts]: Yep

File: src/test/utils.ts (hunk 8 above)
Comment (orta) [reply src/test/utils.ts]: Yes, there is a test that does this by injecting into the gravQL schema - https://github.com/artsy/metaphysics/blob/f9a68d155ffdbdeb832e72394568d7eea3150dd3/src/schema/me/__tests__/record_artwork_view.test.js#L9-L43

**Comment by orta** [comment]
OK, this is good to re-go

**Comment by alloy** [comment]
LGTM 👍

**Pull request merged**